integration-tests/README.md

```diff
@@ -0,0 +1,123 @@
+# Realm JS integration tests
+
+This directory of the repository is contains two sub-directories:
+- [./tests](./tests): A suite of tests which are supposed to pass in all defined environments.
+- [./environments](./environments): A couple of environments in which the tests are supposed pass:
+  - [Node.js](./environments/node/README.md)
+  - [React Native](./environments/react-native/README.md)
+  - [Electron](./environments/electron/README.md)
+
+## Using the integration tests
+
+Before installing the integration test package, ensure that you've installed the Realm JS package:
+
+```bash
+cd ..
+npm install
+cd integration-tests
+```
+
+### Installing the integration test "meta-package"
+
+For convenience a single install script is provided in this directory:
+
+```bash
+npm install
+```
+
+This will first produce a .tgz package of the Realm JS library after which the development dependencies of the test
+suite and each of the environments will be installed. The Electron environment will rebuild realm's native module
+for the Electron runtime.
+
+The environments consume Realm and the test suite as packaged .tgz files to resemble a more life-like scenario and
+to prevent symbolic linking, which can mess with the environments build tools as well as the `package-lock.json`s.
+
+### Installing an environment on CI
+
+When installing on CI we don't want to install all environments for every executing environment, therefore the
+environments support installing individually too.
+
+To install an environment on CI we simply ensure that a `realm-*.tgz` package is available in the `integration-tests`
+directory, change directory to the particular environment and install the package
+
+```bash
+# Ensure a packaged archive
+cd integration-tests
+REALM_BUILD_ANDROID=1 npm pack ..
```

Comment:
No clue - it's read from the pre-packaging script (https://github.com/realm/realm-js/blob/v2.23.0/scripts/prepublish.js#L29) ... but I'll make that more explicit.

Reply:
Otherwise it won't build the Android stuff. The `.so` files used on Android are shipped in the package so you need to build it first. Same trick is used in `scripts/publish.sh`.